deployment/postgres-init-db/init-aerie.sh

```diff
@@ -31,6 +36,12 @@ psql -v ON_ERROR_STOP=1 --username "aerie" --dbname "aerie_merlin" <<-EOSQL
   \echo 'Done!'
 EOSQL
 
+psql -v ON_ERROR_STOP=1 --username "aerie" --dbname "aerie_sched" <<-EOSQL
+  \echo 'Initializing aerie_sched database objects...'
+  \ir /docker-entrypoint-initdb.d/sql/sched/init.sql
```

Comment:
:large_blue_circle: Same here (sched) 😄 . Happy with whatever you choose though. Just asking the question.